tests/mir-opt/jump_threading.assume.JumpThreading.panic-unwind.diff

```diff
@@ -0,0 +1,39 @@
+- // MIR for `assume` before JumpThreading
++ // MIR for `assume` after JumpThreading
+  
+  fn assume(_1: u8, _2: bool) -> u8 {
+      let mut _0: u8;
+  
+      bb0: {
+          switchInt(_1) -> [7: bb1, otherwise: bb2];
+      }
+  
+      bb1: {
+          assume(_2);
+-         goto -> bb3;
++         goto -> bb6;
```

Comment:
Naive question: Shouldn't jump threading recurse and have that jump to bb5?

Reply:
This is possible, but I chose to separate concerns here. Jump threading duplicates blocks to thread jumps, and SimplifyCFG cleans up goto chains after.